handler.cpp

```diff
@@ -5,33 +5,9 @@
 #include <vector>
 #include <string>
 
-echo_handler::echo_handler(std::vector<char>& request) {
```

Comment:
Do we not need this stuff for echo handling? maybe comment it out instead of deleting so we can reuse later?